neo4j/dbtype/graph.go

```diff
@@ -22,18 +22,22 @@ package dbtype
 
 // Node represents a node in the neo4j graph database
 type Node struct {
-	Id     int64                  // Id of this node.
-	Labels []string               // Labels attached to this Node.
-	Props  map[string]interface{} // Properties of this Node.
+	Id        int64                  // Id of this Node.
```

Comment:
Is there a way of depreciating this Id?

Reply:
Yes, but... is it really a deprecation here?
`Id` as `int64` will be gone in 6.0 and replaced by... `Id` as a `string` or am I missing something?

Reply:
For changing the type safely in `6.0`, is it not important making devs do not use the property as an integer first?

Reply:
The problem I see is two-fold:

 - `Id` will be deprecated without a proper alternative **beyond** 5.x - in 6.0, there will be a breaking change no matter what
 - shouldn't `ElementId` be deprecated already since it's gonna be renamed in 6.0?
 
I think deprecations will bring more confusion. 
I agree, however, that I should explicitly document the future plans for `Id` and `ElementId` in the docs.

Reply:
My bad, `ElementId` is here to stay.. Let me add proper docs.

Reply:
fixed in c3df8b9a524be38d11a81a119070f58d63a068c3